src/helpers/tests/flow-chart.testData.ts

```diff
@@ -0,0 +1,205 @@
+export const legacyYAML = `!Flow
+pods:
+  segmenter:
+    show_exc_info: true
+    uses: pods/segment.yml
+    read_only: true
+  encoder:
+    show_exc_info: true
+    uses: pods/encode.yml
+    polling: any
+    shards: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  chunk_idx:
+    polling: any
+    show_exc_info: true
+    uses: pods/chunk.yml
+    shards: $JINA_SHARDS
+  doc_idx:
+    polling: any
+    uses: pods/doc.yml
+    needs: gateway
+  join_all:
+    uses: _merge
+    needs: [doc_idx,chunk_idx]
+`;
+
+export const v1YAML = `!Flow
+version: '1'
+pods:
+  - name: segmenter
+    show_exc_info: true
+    uses: pods/segment.yml
+    read_only: true
+  - name: encoder
+    show_exc_info: true
+    uses: pods/encode.yml
+    polling: any
+    shards: $JINA_PARALLEL
+    timeout_ready: 600000
+    read_only: true
+  - name: chunk_idx
+    polling: any
+    show_exc_info: true
+    uses: pods/chunk.yml
+    shards: $JINA_SHARDS
+  - name: doc_idx
+    polling: any
+    uses: pods/doc.yml
+    needs: gateway
+  - name: join_all
+    uses: _merge
+    needs: [doc_idx, chunk_idx]
+`;
+
+export const parsedLegacyYaml = {};
```

Comment:
You forgot that, right?
Do you alreay use GitHub Desktop. It doesn't let me commit if the linter complains. And the linter complains with unused vars